src/main/java/com/openlattice/data/DataIntegrationApi.java

```diff
@@ -30,60 +30,17 @@
      * These determine the service routing for the LB
      */
 
-    // @formatter:off
-    String SERVICE               = "/datastore";
-    String CONTROLLER            = "/integration";
-    String BASE                  = SERVICE + CONTROLLER;
-    // @formatter:on
-
-    String ASSOCIATION = "association";
-
-    String COUNT                 = "count";
-    String DETAILED_RESULTS      = "detailedResults";
-    String EDGES                 = "edges";
-    String ENTITY_KEY_ID         = "entityKeyId";
-    String ENTITY_KEY_IDS        = "entityKeyIds";
-    String ENTITY_KEY_ID_PATH    = "{" + ENTITY_KEY_ID + "}";
-    /**
-     * To discuss paths later; perhaps batch this with EdmApi paths
-     */
-
-    String ENTITY_SET            = "set";
-    String ENTITY_SET_ID         = "setId";
-    String SET_ID_PATH = "{" + ENTITY_SET_ID + "}";
-    String UPDATE      = "update";
+    String SERVICE          = "/datastore";
```

Comment:
Unfortunately, I don't think these deletions are safe -- I think some old apps/SD microservices are still using these. I'll make tickets to move them all over, but until that's done, I think we need to keep these.

Reply:
Thanks, didn't knew that! Will reverse

Reply:
np! this will be super nice to do someday though... also made a ticket for it on myself

https://jira.openlattice.com/browse/LATTICE-2382